apps/site/assets/ts/schedule/components/line-diagram/LineDiagram.tsx

```diff
@@ -89,21 +87,7 @@ const LineDiagram = ({
     modalOpen: false
   });
 
-  const {
-    data: maybeLiveData,
-    isLoading: liveDataIsLoading,
-    // @ts-ignore https://github.com/async-library/react-async/issues/244
-    reload: reloadLiveData
-  } = useFetch(
-    `/schedules/line_api/realtime?id=${route.id}&direction_id=${directionId}`,
-    {},
-    { json: true, watch: directionId }
-  );
-  const liveData = (maybeLiveData || {}) as LiveDataByStop;
-  useInterval(() => {
-    /* istanbul ignore next */
-    if (!liveDataIsLoading) reloadLiveData();
-  }, 15000);
+  const liveData: LiveDataByStop = {};
```

Comment:
@thecristen maybe I was unclear, but the idea was to remove just the `useInterval` call so that the page will load the `liveData` one time, without repeatedly polling it. @ingridpierre am I remembering our agreement of this morning correctly?

Reply:
Sorry about that, I've updated this accordingly.